examples/map_transformations/reprojection_different_observers.py

```diff
@@ -35,11 +35,11 @@
 # Let’s download an EUV image from both AIA and EUVI A, when the
 # two spacecraft were separated by approximately 120 degrees.
 
-euvi = (a.vso.Source('STEREO_A')&
-        a.Instrument('EUVI') &
+euvi = (a.vso.Source('STEREO_A') &
```

Comment:
I did do `a.vso.Source.stereo` but that seems to be Stereo B and downloaded no data that I could see.